webapp/.env

```diff
@@ -1,7 +1,8 @@
 NEXT_PUBLIC_BTC_INPUTS_SIZE=105
 NEXT_PUBLIC_BTC_OUTPUTS_SIZE=25
 # Once mainnet is published, these values below may go into .env.development as they belong to hemi's testnet
+NEXT_PUBLIC_BITCOIN_NETWORK=testnet
```

Comment:
While this works for now, in the redesigned version this will be decided  from a navbar

<img width="301" alt="image" src="https://github.com/user-attachments/assets/42ace070-08bd-4042-a3ec-c92f63662575">

which after choosing either `testnet` or `mainnet`, updates a query string and shows the appropriate chains from the environment.

This dropdown is not available now in the UI, but the support for multichain already exists in `main`. So you could implement this not to use the `.env` and directly instantiate the appropriate instance depending on this flag.

You can see how to get the appropriate Bitcoin instance in [here](https://github.com/hemilabs/ui-monorepo/blob/main/webapp/hooks/useBitcoin.ts).

The bitcoin chains definitions are [here](https://github.com/hemilabs/ui-monorepo/blob/main/btc-wallet/chains/index.ts) - there's a `testnet` flag if you need to `if/else` somewhere given a bitcoin chain

Considering this wasn't already supported (the btc API was hardcoded to testnet), _this is ok and can be merged_, but you want to create an issue and link it in the comments or something.

Reply:
@gndelia I think we should add that support in a different PR as `utils/btcApi.ts` never had a way to switch networks nor this PR introduces that.

To allow switching networks, `utils/btcApi.ts` may need to export a function that receives the current network name and returns an object with the utility functions. Other option is to add a `network` parameter to all calls. Either way I guess it is a big change worth doing in a different PR.

Reply:
sounds ok for me, can you just create an issue so we don't forget about that?

Reply:
https://github.com/hemilabs/ui-monorepo/issues/557